cloudsql-mysql-plugin/docs/CloudsqlMysql-batchsink.md

```diff
@@ -0,0 +1,155 @@
+# CloudSQL MySQL Batch Sink
+
+
+Description
+-----------
+Writes records to a CloudSQL MySQL table. Each record will be written to a row in the table.
+
+
+Use Case
+--------
+This sink is used whenever you need to write to a MySQL table.
+Suppose you periodically build a recommendation model for products on your online store.
+The model is stored in a FileSet and you want to export the contents
```

Comment:
Can you remove the reference to FileSet? You can use GCS instead.

Reply:
Changed to GCS.